examples/generate_bom/compute_cogs.py

```diff
@@ -115,7 +115,11 @@ def fetch_price_for_part(part_number: str) -> dict[int, float]:
     )
     parser.add_argument(
         "--output-file",
-        help="The path to the output file. Defaults to stdout, i.e. printing to the console.",
+        help="The relative path to the output file. Defaults to stdout, i.e. printing to the console.",
```

Comment:
Can we mention that this is relative to the workspace directory passed in?

Its not clear to me why we need a workspace directory and the current working directory won't work.

Reply:
I agree, I would prefer the user of the script deal with setting up the working directory.